### ISSUE TITLE
[Helm] Function dig not defined

### ISSUE THREAD

**Issue by abhi1693** [minio/operator#811]
## Expected Behavior

Should be able to install minio-operator with minimum configuration

## Current Behavior

```
Wait helm template failed. Error: parse error at (minio-operator/templates/tenant.yaml:11): function "dig" not defined Use --debug flag to render out invalid YAML : exit status 1
```

## Possible Solution

Maybe add the `dig` function to the `_helpers.tpl`. I'm not sure if that's the correct solution.

## Steps to Reproduce (for bugs)

1. Using v4.2.5, deploy the app using the following configuration
```
---
console:
  ingress:
    enabled: true
    host: sslip.io
tenants:
  # Tenant name
  - name: minio
    namespace: minio
    pools:
      - servers: 1
        volumesPerServer: 1
        size: 5Gi
        storageClassName: longhorn
    secrets:
      # create a kubernetes secret object with the accessKey and secretKey as defined here.
      enabled: true
      name: minio-secret
      accessKey: minio
      secretKey: minio123
```

## Context

I'm trying to test the operator so that I can transition my workload from the legacy minio helm chart

## Your Environment

* Version used (`minio-operator`): 4.2.5
* Environment name and version (e.g. kubernetes v1.17.2): rancher v2.5.9, kubernetes v1.20.4
* Server type and version: Ubuntu 20.04

**Comment by harshavardhana** [comment]
Dig function is supported by new sprig, please update your rancher fleet version.

**Comment by Splaktar** [comment]
I ran into this running the instructions in https://github.com/minio/operator/blob/master/helm/minio-operator/README.md.

I don't know what sprig or rancher are, unfortunately. I'm using K8s via GKE 1.20.8-gke.2100.

**Comment by Splaktar** [comment]
I resolved this by updating to Helm v3.6.3.

**Comment by harshavardhana** [comment]
> I resolved this by updating to Helm v3.6.3.

This is the correct resolution.
